dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -8,13 +8,21 @@
 ! TSA configuration
 !
 ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERFACE, "Loopback0") | ip }}/32
+{% if multi_npu()  == true %}
+  ip prefix-list PL_LoopbackV4 permit {{ get_ipv4_loopback_address(LOOPBACK_INTERFACE, "Loopback1") | ip }}/32
```

Comment:
There are currently discussions on adding multiple loopback interfaces to Sonic and if there is a regular "Loopback1", say got added from previous config_db etc, it could result in some conflicts. My suggestion is, can we have it renamed to something specific?

Reply:
@prsunny , changed from Loopback1 to Loopback4096 for ASIC's loopback interface